fv3core/tests/savepoint/test_translate.py

```diff
@@ -369,28 +368,42 @@ def test_parallel_savepoint(
 
 
 def save_netcdf(
-    testobj, inputs_list, output_list, ref_data, failing_names, out_filename
+    testobj,
+    # first list over rank, second list over savepoint
+    inputs_list: List[Dict[str, List[np.ndarray]]],
+    output_list: List[Dict[str, List[np.ndarray]]],
+    ref_data: Dict[str, List[np.ndarray]],
+    failing_names,
+    out_filename,
 ):
     import xarray as xr
 
     data_vars = {}
     for i, varname in enumerate(failing_names):
-        dims = [dim_name + f"_{i}" for dim_name in testobj.outputs[varname]["dims"]]
-        attrs = {"units": testobj.outputs[varname]["units"]}
+        if hasattr(testobj, "outputs"):
+            dims = [dim_name + f"_{i}" for dim_name in testobj.outputs[varname]["dims"]]
+            attrs = {"units": testobj.outputs[varname]["units"]}
+        else:
+            dims = [
+                f"dim_{varname}_{j}" for j in range(len(ref_data[varname][0].shape))
+            ]
```

Comment:
Do these two have the same dimension name? I think it would be nice to have the same variable (computed, reference, and error) use the same dimension name, it would make doing basic operations easy.